scripts/delete.js

```diff
@@ -1,24 +1,24 @@
 #!/usr/bin/env node
 
-require(`./utils/protectStage`)();
+require('./utils/protectStage')();
 
 // Dependencies
 const deleteServerlessService = require('./utils/deleteServerlessService');
 
 const services = [
-  `storage-objects`,
-  `ingestion-etl-results`,
-  `storage-signed-uploads`,
-  `storage-deleter`,
-  `storage-meta-index`,
-  `harmonized-storage`,
-  `ingestion-manager`,
-  `ingestion-cleaner`,
-  `ingestion-etl-agri-csv`,
-  `ingestion-etl-budg-xls`,
-  `ingestion-etl-inforegio-json`,
-  `ingestion-etl-inforegio-xml`,
-  `value-store-projects`,
+  'storage-objects',
```

Comment:
Is there any problem with this syntax?

Reply:
Only that template literals are not needed here. I tend to always choose the simplest solution 😋